receiver/windowsperfcountersreceiver/testdata/config-emptyinstance.yaml

```diff
@@ -1,10 +1,16 @@
 receivers:
   windowsperfcounters:
+    metrics:
+      metric: 
+        description: desc
+        unit: "1"
+        gauge:
```

Comment:
I think we are missing cases where either `sum` is specified here, or nothing.

Reply:
Added two more test cases